src/Agent.Worker/Build/UploadResult.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.VisualStudio.Services.Agent.Worker.Build
+{
+    public class UploadResult
```

Comment:
add a default constructor to initialize FailedFiles=new list<string>() and TotalFileSize=0, this can avoid null ref exception for consumer.

Reply:
Added constructors to this class